src/contracts/pods/EigenPod.sol

```diff
@@ -757,12 +757,9 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
     }
 
     function _calculateValidatorPubkeyHash(bytes memory validatorPubkey) internal view returns(bytes32){
-         bytes memory paddedPubkey = new bytes(validatorPubkey.length + 16);
-
-        // Copy original data to the new array
-        for (uint i = 0; i < validatorPubkey.length; i++) {
-            paddedPubkey[i] = validatorPubkey[i];
-        }
+        require(validatorPubkey.length == 48, "EigenPod._calculateValidatorPubkeyHash must be a 48-byte BLS public key");
+        bytes16 zeroPadding = bytes16(0);
+        bytes memory paddedPubkey = abi.encodePacked(validatorPubkey, zeroPadding);
         return sha256(paddedPubkey);
```

Comment:
just `return sha256(abi.encodePacked(validatorPubkey, bytes16(0)))` is cleaner

Reply:
dawg just tryna get me insertions up